src/Symfony/Bundle/FrameworkBundle/Command/EventDispatcherDebugCommand.php

```diff
@@ -33,7 +33,7 @@ protected function configure()
         $this
             ->setName('debug:event-dispatcher')
             ->setDefinition(array(
-                new InputArgument('event', InputArgument::OPTIONAL, 'An event name (foo)'),
+                new InputArgument('event', InputArgument::OPTIONAL, 'An event name (kernel.request)'),
```

Comment:
We don't provide such examples anywhere else for command arguments. Example should be in the description of the command instead.